firestone/enUS.json

```diff
@@ -202,7 +202,12 @@
 			"windfury": "Windfury",
 			"mega-windfury": "Mega Windfury"
 		},
-		"duration": "{{minutes}}:{{seconds}} min"
+		"duration": {
+			"min-sec": "{{min}}:{{sec}} min",
+			"hrs-min": "{{hrs}} hours {{min}} min",
+			"min": "{{min}} min",
+			"sec": "{{sec}} s"
```

Comment:
Do we have use cases for these other formats today? If not, maybe we can keep the same translation key, and add the new formats when we need them.

Reply:
Of course, this is not just done. This is the first step towards the localization of this block.
![2022-08-31_03-27-36](https://user-images.githubusercontent.com/43519401/188472955-b8b7d2ea-d376-4077-a248-c405ab9fd5ba.png)